app/models/custom_attributes/date_time_value.rb

```diff
@@ -9,7 +10,11 @@ def value=(newVal)
     end
 
     def value
-      self.date_time_value
+      format_date_input(date_time_value, Company.find(company_id))
```

Comment:
gem should not be dependant on EZO - write a method here to format the date with the format passed as in argument